src/core/optimization_container.jl

```diff
@@ -856,12 +856,13 @@ function add_param_container!(
     name::String,
     axs...;
     meta = CONTAINER_KEY_EMPTY_META,
+    time_series_multiplier_id = 1,
```

Comment:
we should default to `1` when making the attributes and modify the attribute at the `add_param_container!` call site.

Reply:
so you're saying that this function should be?

```
function add_param_container!(
    container::OptimizationContainer,
    ::T,
    ::Type{U},
    ::Type{V},
    name::String,
    axs...;
    meta = CONTAINER_KEY_EMPTY_META,
) where {T <: TimeSeriesParameter, U <: PSY.Component, V <: PSY.TimeSeriesData}
    param_key = ParameterKey(T, U, meta)
    if isabstracttype(V)
        error("$V can't be abstract: $param_key")
    end
    attributes = TimeSeriesAttributes(V, name)
    return _add_param_container!(container, param_key, attributes, axs...)
end
```

Reply:
Yes, it should be unchanged and then we manipulate the attribute at the call site.